src/app/auth/change_password.tsx

```diff
@@ -0,0 +1,55 @@
+import { Redirect, router } from 'expo-router';
+import React, { useState } from 'react';
+import { Alert, View } from 'react-native';
+import { Button, Input } from 'react-native-elements';
+
+import globalStyles from '../../styles/globalStyles';
+import { useSession } from '../../utils/AuthContext';
+
+function VerificationScreen() {
+  const { updateUser, user } = useSession();
+  const [password, setPassword] = useState('');
+  const [loading, setLoading] = useState(false);
+
+  const changePassword = async () => {
+    setLoading(true);
+    const { error } = await updateUser({ password });
+
+    if (error) {
+      console.error(error);
+      Alert.alert('Updating password failed');
+    } else {
+      router.replace('/home');
+    }
+
+    setLoading(false);
+  };
+
+  if (!user) return <Redirect href="/auth/login" />;
+
+  return (
+    <View style={globalStyles.auth_container}>
+      <View style={globalStyles.verticallySpaced}>
```

Comment:
Please add an input for the current password. A user should have to input the current password correctly in order to reset it.